web/sites/default/config/webform.webform.report_park_maintenance_issue.yml

```diff
@@ -380,8 +385,12 @@ elements: |-
       report_photo:
         '#type': webform_image_file
         '#title': Photo
-        '#description': 'If appropriate, please include a photo of the issue you are reporting. Please do not include photos where other people are present and identifiable.'
+        '#description': '<p>If appropriate, please include a photo of the issue you are reporting. Please do not include photos where other people are present and identifiable.</p>'
         '#description_display': before
+        '#states':
+          invisible:
+            ':input[name="what_are_you_trying_to_report"]':
+              value: 'Feedback about a staff encounter'
         '#file_preview': 'medium:image'
         '#max_filesize': '30'
         '#file_extensions': 'gif jpg png svg jpeg'
```

Comment:
Not something you changed as part of the PR, but I wonder if we should remove GIF and SVG from here?

Reply:
Also good call about the file types. I  can't imagine someone uploading a GIF.